doc/contributing/releases.md

```diff
@@ -249,6 +249,15 @@ each commit from the PR. You will also need to manually apply the PR
 metadata (`PR-URL`, `Reviewed-by`, etc.) by amending the commit messages. If
 known, additionally include `CVE-ID: CVE-XXXX-XXXXX` in the commit metadata.
 
+Push the commits to the `vN.x.x-proposal` branch that has branched off of
+`vN.x`.
+
+```console
+git checkout vN.x
+git checkout -b vN.x.x-proposal
+git cherry-pick  ...  # cherry-pick nodejs-private PR commits directly into the proposal
+```
```

Comment:
This content does seem to duplicate what is mentioned above in lines 239 / 240. I think it might make sense to move the content related to picking a branch name there. It might be good to explicitly call out being careful when pushing, as pushing to the wrong branch can result in early release of embargoed information.

Reply:
Fixes. I've also included the warning to not push to the wrong remote.